parsers/test/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from requests import Session
+from requests_mock import Adapter
+
+
+@pytest.fixture
+def adapter():
+    adapter = Adapter()
+    yield adapter
+
+
+@pytest.fixture
+def session(adapter):
+    session = Session()
+    session.mount("https://", adapter)
+    yield session
```

Comment:
Could we do this for HTTP too?

I'm not sure we can guarantee that everything will be HTTPS all the time.

Reply:
True true, i didn't feel confident about the impact of mount, so i went for https nonstop instead and transktioned two tests to work against an https endpoint as it also worked in those cases.

Do you think it is fine to do a http:// alongside here without a negative consequence do you think?

I'm leaning towards adding it if its needed in the future, but adding a comment about it for now saying that it hasnt been needed so far, but that it probably can be added whenever it is

Reply:
I'll read up and trial a bit!

Reply:
We have other adapters for non test purposes that does it and they seem to work. You can take a look here if you wish:
https://github.com/electricitymaps/electricitymaps-contrib/blob/9e8130fe2207d85453421babad8a5d5dac2aa55c/parsers/lib/session.py#L16-L22

So I think is should be fine!